tests/features/pages/record-page.feature

```diff
@@ -66,10 +66,10 @@ Feature: Record page
 
   Scenario: IIIF Image viewer
     When I open a `record page with a IIIF Image`
-    Then I see the `IIIF Image viewer`
+    Then I don't have the `IIIF viewer`
```

Comment:
this expectation is wrong, as it'll show the IIIF viewer.

Reply:
Indeed. It's been corrected.